packages/node_modules/@node-red/nodes/core/function/90-exec.js

```diff
@@ -122,12 +123,14 @@ module.exports = function(RED) {
                     });
                 }
                 else {
+                    var execOpt = {encoding:'binary', maxBuffer:10000000};
+					if (process.platform === 'linux' && fs.existsSync('/bin/bash')) { execOpt.shell = '/bin/bash'; }
```

Comment:
If we want to go done this route, can you move the `fs.existsSync` check to either the node constructor or even at the top level of the module - there's no need to reevaluate it every time the node is triggered.